config/inventory.ini

```diff
@@ -2,26 +2,31 @@
 # ## different ip than the default iface
 # ## We should set etcd_member_name for etcd cluster. The node that is not a etcd member do not need to set the value, or can set the empty string value.
 [all]
-${PREFIX}-node1 ansible_user=ubuntu ansible_host=95.54.0.12 etcd_member_name=etcd1 # Master 1
-${PREFIX}-node2 ansible_user=ubuntu ansible_host=95.54.0.13
-${PREFIX}-node3 ansible_user=ubuntu ansible_host=95.54.0.14
+${PREFIX}-node1 ansible_user=ubuntu ansible_host=<public ip> ip=<private ip> etcd_member_name=etcd1 # Master 1
+${PREFIX}-node2 ansible_user=ubuntu ansible_host=<public ip> ip=<private ip> etcd_member_name=etcd2 # Master 2
+${PREFIX}-node3 ansible_user=ubuntu ansible_host=<public ip> ip=<private ip> etcd_member_name=etcd3 # Master 3
+${PREFIX}-node4 ansible_user=ubuntu ansible_host=<public ip> ip=<private ip>
+${PREFIX}-node5 ansible_user=ubuntu ansible_host=<public ip> ip=<private ip>
+${PREFIX}-node6 ansible_user=ubuntu ansible_host=<public ip> ip=<private ip>
```

Comment:
Could you also add `access_ip`, or at least a comment with [this](https://kubespray.io/#/docs/vars?id=addressing-variables):
> access_ip - IP for other hosts to use to connect to. Often required when deploying from a cloud, such as OpenStack or GCE and you have separate public/floating and private IPs.